js/script.js

```diff
@@ -1,11 +1,9 @@
-
 function getTriangleArea(a, h) {
-  if (a === 0) {
+  if (a > 0 && h > 0) {
+    return a * h / 2;
+  } else {
```

Comment:
else is unnecessary, because return in previous if statement stop function when conditions will be true, so just return message that data are wrong without else ;)